### ISSUE TITLE
Create SensorMath module

### ISSUE THREAD

**Issue by thareUSGS** [USGS-Astrogeology/SensorUtils#96]
This should have the rec2lat and lat2rec methods in it

**Comment by kberryUSGS** [comment]
sensormath namespace

- [ ] angle(ray1: CartesianVector, ray2: CartesianVector): double
- [ ] distance(point1: CartesianPoint, point2: CartesianPoint): double

- [ ] cartesianToVec(CartesianPoint): arma::vec
- [ ] imageToVec(ImagePoint): arma::vec
- [ ] vecToCartesian(arma::vec): CartesianPoint
- [ ] vecToImage(arma::vec): ImagePoint
     
(Not required, but may updated since this was previously existing in `SensorUtils.cpp`):
- [ ] lat2rec modify to use updated Points
- [] rec2lat modify to use updated Points

**Comment by ihumphrey-usgs** [comment]
@kberryUSGS Could you update the SensorMath.h header with the signatures for the functions in SensorMath.cpp? (if you are working on this still)

**Comment by ihumphrey-usgs** [comment]
Also, would it be useful to define a ```pi()``` function in the sensormath library, so it could either use c/c++'s **M_PI**, arma's **datum::pi**, or any other implementation?

**Comment by kberryUSGS** [comment]
@ihumphrey-usgs Yes, can do (but probably tomorrow morning)! Those are both good ideas.

**Comment by kberryUSGS** [comment]
For documentation this decision's sake: We decided in the meeting this morning to just use M_PI from cmath for pi.
